dttools/src/disk_allocator.c

```diff
@@ -10,16 +10,53 @@ See the file COPYING for details.
 #include <inttypes.h>
 #include <unistd.h>
 #include <string.h>
-
+#include "getopt.h"
 #include "disk_alloc.h"
 #include "stringtools.h"
+#include "cctools.h"
+
+static void show_help(const char *cmd) {
+
+	fprintf(stdout, "Use: %s [options]\n", cmd);
```

Comment:
This needs to inform what the required arguments are. I believe you require 3: `func`, `loc`, and `fs`?